cppapi/client/group.h

```diff
@@ -1044,15 +1044,33 @@ class Group : public GroupElement
   std::vector<std::string> get_device_list (bool fwd = true);
   //-
 
+/**
+ * Enables group element
+ *
+ * Enables group element. The element will participate in all group operations.
+ *
+ * @param [in] device_name name of the element, can contain wildcards (*).
+ * If more than one device matches the pattern, only the first one enabled.
```

Comment:
"only the first one enabled." => "only the first one is enabled." or "only the first one will be enabled."
Same comment for the disable method.

Reply:
THanks! Fixes as proposed